lib/openstack_query/query_executer.py

```diff
@@ -97,9 +111,12 @@ def run_query(
             runner of interest.
         """
 
+        if isinstance(cloud_account, str):
+            cloud_account = CloudDomains.from_string(cloud_account)
```

Comment:
This still would force us to use either `PROD` or `DEV`?

Reply:
I'm not comfortable removing it - I'll leave it for another PR